src/Aspire.Dashboard/Components/Pages/Resources.razor

```diff
@@ -77,6 +80,13 @@
                                       Title="@ControlsStringsLoc[nameof(ControlsStrings.ViewAction)]"
                                       OnClick="() => ShowResourceDetails(context)">@ControlsStringsLoc[nameof(ControlsStrings.ViewAction)]</FluentButton>
                     </TemplateColumn>
+                    @if (HasResourcesWithCommands)
+                    {
+                        <TemplateColumn Title="@Loc[nameof(Dashboard.Resources.Resources.ResourcesCommandsColumnHeader)]">
+                            <ResourceCommands Commands="@context.Commands"
+                                              CommandSelected="async (command) => await ExecuteResourceCommandAsync(context, command)" />
+                        </TemplateColumn>
+                    }
```

Comment:
There were issues with the metrics table and dynamic columns like this. Basically, the table data would change but it would take a moment before the columns updated. I don't think that's an issue here, but something to keep an eye on.